.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**What is the goal of the feature?**
+A clear and concise description of what you want to happen.
```

Comment:
I think it's still an XY Problem
A sentence that looks like "Describe the goal of the feature" or "Describe what the feature should do" is more appropriated